crates/sdk/src/network/auth.rs

```diff
@@ -1,142 +0,0 @@
-use std::{borrow::Cow, str::FromStr};
```

Comment:
note: I am nuking v1 and I will convert `network-v2` to just `network` (after review because otherwise diff would be nightmarish)